src/TodoListItem.js

```diff
@@ -1,7 +1,7 @@
-import React from "react";
+import React from 'react';
 
-const TodoListItem = (props) => {
-  return <li>{props.todo}</li>;
+const TodoListItem = ({ id, todo }) => {
+  return <li key={id}>{todo}</li>;
```

Comment:
Oh here's a spot that does require a small change. You don't need the `id` here at all. The `<li>` is rendered inside of the `TodoListItem` component which receives the `id` as a `key` when it is rendered by the `TodoList` component. 

```suggestion
const TodoListItem = ({ todo }) => {
  return <li>{todo}</li>;
```